app/src/main/java/com/example/fittracker/MainActivity.java

```diff
@@ -1,45 +1,33 @@
 package com.example.fittracker;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-import android.view.View;
-import android.widget.Button;
-import android.widget.EditText;
+import android.support.v7.app.AppCompatActivity;
+
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 
 public class MainActivity extends AppCompatActivity {
     private LogInContract.Presenter presenter;
-    private EditText mail;
-    private EditText password;
-    private UserBase lista;
-    private final String MAIL_PRUEBA = "danko94.cg@gmail.com";
-    private final String PASS_PRUEBA = "1234";
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        mail=findViewById(R.id.mail_usuario);
-        password=findViewById(R.id.password);
-        lista = new UserBase();
-        lista.addUsuario(new User(MAIL_PRUEBA,PASS_PRUEBA));
+        ButterKnife.bind(this);
         init();
     }
-    public void init(){
-        LogInContract.View view = new MainActivityView(mail,password,this);
-        LogInContract.Model model = new MainActivityModel(lista);
+
+    public void init() {
+        LogInContract.View view = new MainActivityView(this);
+        LogInContract.Model model = new MainActivityModel();
         presenter = new MainActivityPresenter(view, model);
-        final Button logIn = findViewById(R.id.log_in);
-        final Button signUp = findViewById(R.id.sign_up);
-        logIn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                presenter.onLogInPressed();
-            }
-        });
-        signUp.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                presenter.onSignUpPressed();
-            }
-        });
+        @OnClick(R.id.log_in)
```

Comment:
put the logIn and signUp methods outside the init()

Reply:
done